### ISSUE TITLE
Einzelne Veranstaltungen

### ISSUE THREAD

**Issue by fredericaudio** [disktree/stritter.audio#6]
Evtl. Pop ups oder ähnliches für einzelne Veranstaltungen 
- mit Flyer, Fotos oder anderen spez. Infos

**Comment by tong** [comment]
Kann man machen, aber vielleicht nochmal überlegen, weil es auch brauchbar wäre die auf einzelnen sites zu haben damit man einen url hat den man auf social media posten kann und als email verschicken usw.

Eventuell sonst beides machen und die einezelnen sites, wie gehabt, nirgends verlinken. Dann kann mans trotzdem verschicken/posten aber hat keine leer aussehenden sites bei denen es keine infos/bilder gibt.

**Comment by fredericaudio** [comment]
ja, das wäre auch eine Lösung. Dann also unter den Dates, ein paar Flyer/Fotos und Info.

schwierig nur dann, wenn es keinen eigenen Projekte Reiter für eine Veranstaltung gibt....
